### ISSUE TITLE
torch 警告

### ISSUE THREAD

**Issue by WangshilongChina** [LuminosityX/HAT#4]
"Seems like `optimizer.step()` has been overridden after learning rate scheduler "
                              "initialization. Please, make sure to call `optimizer.step()` before "
                              "`lr_scheduler.step()`. See more details at "
                              "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate" 你好作者，我在训练过程中出现了这个警告，报错路径是torch包（使用cu111_1.8.1_torch）,请问作者当时也出现这种情况了吗？是否对训练造成影响
